Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.14.4 as builder
```

Comment:
- It is new project; start  with GO 1.15 (DockerFile, mod files)

Reply:
done